crates/router/src/core/api_keys.rs

```diff
@@ -12,13 +13,45 @@ use crate::{
     utils,
 };
 
+pub static HASH_KEY: tokio::sync::OnceCell<StrongSecret<[u8; PlaintextApiKey::HASH_KEY_LEN]>> =
+    tokio::sync::OnceCell::const_new();
+
+pub async fn get_hash_key(
+    api_key_config: &settings::ApiKeys,
+) -> errors::RouterResult<StrongSecret<[u8; PlaintextApiKey::HASH_KEY_LEN]>> {
+    #[cfg(feature = "kms")]
+    let hash_key = crate::services::kms::KeyHandler::get_kms_decrypted_key(
```

Comment:
remove absolute path here

Reply:
Then that import would end up behind the `kms` feature flag, since this is the only place where this function is used. Would you prefer that?

Reply:
Do the same in the import place itself so the we follow one approach of importing.